src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] ggss) {
+
+        Machine[] machines = {new Truck(), new Excavator(), new Bulldozer()};
+
+        for(int i = 0; i < machines.length; i++) {
+            machines[i].doWork();
+            machines[i].stopWork();
+        }
```

Comment:
It's recommended to use the enhanced for loop when iterating over an array or a collection. It's more readable and less error-prone. Here is how you can do it: `for (Machine machine : machines) { machine.doWork(); machine.stopWork(); }`.